README.md

```diff
@@ -7,7 +7,7 @@
 [![Code Quality Checks](https://github.com/lmc-eu/spirit-design-system/actions/workflows/test.yaml/badge.svg?branch=main)](https://github.com/lmc-eu/spirit-design-system/actions)
 [![Coverage Status](https://coveralls.io/repos/github/lmc-eu/spirit-design-system/badge.svg?branch=main)](https://coveralls.io/github/lmc-eu/spirit-design-system?branch=main)
 
-Spirit is an open-source design system developed by [LMC][lmc].
+Spirit is an open-source design system developed by [Alma Career (formerly LMC)][alma-career].
```

Comment:
can we add the `formerly` part everywhere where we have this phrase? `developed by [Alma Career][alma-career].`

Reply:
Ok, fixed.